docs/advanced/transform-content.mdx

```diff
@@ -19,7 +19,7 @@ Say we have the following file, `example.mdx`:
 import Component from 'Component';
 
 export const meta = {
-    prop: 'value'
+prop: 'value'
```

Comment:
What’s happening here? 🤔

Reply:
It should be `mdx` but the original type is `md`.

Reply:
@wooorm fixed in https://github.com/mdx-js/mdx/pull/927/commits/a5a4fe4bb5973dd24d790a74604c7849e0924f0b